xml/dm_registers.xml

```diff
@@ -403,7 +453,9 @@
         </field>
         <field name="0" bits="23:13" access="R" reset="0" />
         <field name="busy" bits="12" access="R" reset="0">
-            1: An abstract command is currently being executed.
+            <value v="1" name="busy">
```

Comment:
Should there also be a "ready" value like with authbusy?

Reply:
I've added it. In my change I decided to only modify existing cases of `<number>:` which is why here I only did 1.